HIP/hip-980.md

```diff
@@ -0,0 +1,120 @@
+---
+hip: 980
+title: Enhance mirror node rest APIs with timestamp query parameter
+author: Ivan Kavaldzhiev <ivan.kavaldzhiev@limechain.tech>
+working-group: Steven Sheehy <@steven-sheehy>, Nana Essilfie-Conduah <nana@swirldslabs.com>
+type: Standards Track
+category: Mirror
+needs-council-approval: Yes
+status: In review
+created: 2024-06-03
+requested-by: Solo Labs
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/981
+updated: 2024-06-25
+---
+
+## Abstract
+
+This HIP enhances part of the mirror node REST APIs to support querying by timestamp. This will allow users to query for the historical state of an entity on the network.
```

Comment:
```suggestion
This HIP enhances part of the mirror node REST APIs to support querying by timestamp. This will allow users to query for the historical state of any entity on the network.
```

Reply:
Fixed